.github/workflows/run-tests.yml

```
@@ -14,11 +14,11 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         php: [7.4, 8.0]
-        laravel: [8.*]
+        laravel: [8.*, 9.*]
         stability: [prefer-lowest, prefer-stable]
         include:
-          - laravel: 8.*
-            testbench: ^6.6
+          - laravel: 9.*
+            testbench: ^7.0
```

Please keep the original Laravel 8.* include and replicate it for laravel 9 as I'd like to keep supporting both versions at once.
```suggestion
          - laravel: 8.*
            testbench: ^6.6
          - laravel: 9.*
            testbench: 7.*
```